### PR TITLE
docs: remove schedule

### PR DESCRIPTION
**Pull request by StephenFluin**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit

## PR Type
What kind of change does this PR introduce?

- [x] Documentation content changes

This should be merged to master, 9.0, and 8.x.

### CODE REVIEW

**Review by gkalpak — APPROVED** [review]

File: aio/content/guide/releases.md [hunk 1]
```diff
@@ -84,6 +90,7 @@ This cadence of releases gives you access to new features as soon as they are re
 
 
 
+<<<<<<< HEAD
```
Comment (gkalpak) [inline aio/content/guide/releases.md]: 😱

File: aio/content/guide/releases.md [hunk 2]
```diff
@@ -105,6 +112,8 @@ The following table contains our current target release dates for the next two m
 
 
 
+=======
+>>>>>>> docs: remove schedule
```
Comment (gkalpak) [inline aio/content/guide/releases.md]: 😱 
😱

**Review by gkalpak — CHANGES_REQUESTED** [review]

File: aio/content/guide/releases.md [hunk 3]
```diff
@@ -84,27 +90,6 @@ This cadence of releases gives you access to new features as soon as they are re
 
 
 
-{@a schedule}
```
Comment (gkalpak) [inline aio/content/guide/releases.md]: This was being linked to from a couple of places (so at least we should remove those links):
- https://github.com/angular/angular/blame/4ca7818ce/aio/content/guide/releases.md#L130
- https://github.com/angular/angular/blame/4ca7818ce/aio/content/guide/updating.md#L72

**Review by gkalpak — DISMISSED** [review]
:grin:

**Review by IgorMinar — APPROVED** [review]
please rebase and update the commit message with the justification:

```
Removing for now, since the info is stale, and we need to determine how to adjust the schedule due to the unexpected and significant version 9 delay.
```

**Comment by IgorMinar** [comment]
I made the updates + rebased

**Comment by kara** [comment]
Should be on 8.2.x now with https://github.com/angular/angular/commit/72c3ebe682ef0a12e9b1f725d85bfb79fd23f4e5

**Pull request closed without merging**